package/yast2-network.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Feb 19 11:52:25 UTC 2020 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Add a class to represent NTP servers (jsc#SLE-7188).
```

Comment:
I'm not sure if like the idea about this class in network module. Initially it sounds more for ntp-client class. So, need to check the needs about obtaining the dhcp ntp servers from CaaSP and  from the installer before doing a proper review, so maybe @lslezak and @mchf could have a better opinion about it.

Reply:
Yes, I thought pretty much the same. But, to be honest: 1) yast2-network already contains NTP specific code and 2) I did not want to put another dependency (on yast2-ntp-client) in the related packages. Anyway, I am fine with moving to yast2-ntp-client if required.

Reply:
I think that the benefits far outweighs the risks, so, it is fine with me.